libraries/AP_Logger/README.md

```diff
@@ -68,6 +68,7 @@ Please keep the names consistent with Tools/autotest/param_metadata/param.py:33
 | 's' | "s" | seconds|
 | 'q' | "rpm" | revolutions per minute|  Not an SI unit, but sometimes more intuitive than Hertz|
 | 'r' | "rad" | radians|
+| 't' | "N.m" | Newton meters | torque |
```

Comment:
This should be "N-m" or "N m"

Reply:
Yeah, I agree, it's just matched the existing ones.
![SmartSelect_20241125_142833_Chrome](https://github.com/user-attachments/assets/7ce7fac3-1f2b-4f51-9552-ece7b1f0aa40)